### ISSUE TITLE
Array out of bounds exception

### ISSUE THREAD

**Issue by dkowis** [skipoleschris/xsbt-cucumber-plugin#27]
Instead of getting pretty cucumber output, I get an index out of bounds exception :(

My project: https://github.com/dkowis/turbo-jug-robot

```
$ sbt cucumber
[info] Loading global plugins from /home/dkowis/.sbt/plugins
[info] Loading project definition from /home/dkowis/gitwork/personal/turbo-jug-robot/project
[info] Set current project to turbo-jug-robot (in build file:/home/dkowis/gitwork/personal/turbo-jug-robot/)
[info] No sbt-js sources requiring compilation
[info] Running cucumber...
[error] Exception in thread "main" java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
[error]         at java.util.ArrayList.rangeCheck(ArrayList.java:604)
[error]         at java.util.ArrayList.get(ArrayList.java:382)
[error]         at gherkin.formatter.PrettyFormatter.indentedLocation(PrettyFormatter.java:142)
[error]         at gherkin.formatter.PrettyFormatter.printStep(PrettyFormatter.java:255)
[error]         at gherkin.formatter.PrettyFormatter.match(PrettyFormatter.java:179)
[error]         at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[error]         at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
[error]         at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[error]         at java.lang.reflect.Method.invoke(Method.java:601)
[error]         at cucumber.runtime.Utils$1.call(Utils.java:44)
[error]         at cucumber.runtime.Timeout.timeout(Timeout.java:12)
[error]         at cucumber.runtime.Utils.invoke(Utils.java:40)
[error]         at cucumber.runtime.RuntimeOptions$2.invoke(RuntimeOptions.java:138)
[error]         at com.sun.proxy.$Proxy1.match(Unknown Source)
[error]         at cucumber.runtime.Runtime.runStep(Runtime.java:250)
[error]         at cucumber.runtime.model.StepContainer.runStep(StepContainer.java:49)
[error]         at cucumber.runtime.model.StepContainer.runSteps(StepContainer.java:43)
[error]         at cucumber.runtime.model.CucumberScenario.run(CucumberScenario.java:36)
[error]         at cucumber.runtime.model.CucumberFeature.run(CucumberFeature.java:112)
[error]         at cucumber.runtime.Runtime.run(Runtime.java:105)
[error]         at cucumber.runtime.Runtime.run(Runtime.java:93)
[error]         at cucumber.api.cli.Main.run(Main.java:20)
[error]         at cucumber.api.cli.Main.main(Main.java:12)
java.lang.RuntimeException: There were failed tests.
```

**Comment by skipoleschris** [comment]
Hi,
This is a know bug in the version of gherkin that shipped with the versions of cucumber that the plugin currently uses. This has been fixed and I'm just waiting until the next cucumber release so that I can update the plugin.

**Comment by dkowis** [comment]
Do you perhaps have the link to the issue? I can probably help get it released or release it myself :) I'm actually on that team heh.

**Comment by flurdy** [comment]
Presume it is this Pretty Formatter bug in 1.1.3: https://github.com/cucumber/gherkin/pull/252

**Comment by dkowis** [comment]
Yeah I've been bothering them to get it released, I can't seem to get gherkin to build all it's artifacts on my linux box :(

**Comment by dkowis** [comment]
Cucumber-jvm 1.1.4 is out that should resolve this :)

**Comment by skipoleschris** [comment]
Excellent. I will try to work on releasing a compatible version of the plugin this week. 

Chris Turner

**Comment by skipoleschris** [comment]
Fixed in 0.8.0 release that upgrades to cucumber-jvm 1.1.4
